pkg/ebpf/events_pipeline.go

```diff
@@ -46,6 +46,12 @@ func (t *Tracee) handleEvents(ctx context.Context) {
 	eventsChan, errc = t.processEvents(ctx, eventsChan)
 	errcList = append(errcList, errc)
 
+	eventsChan, errc = t.processTree.StartProcessingPipeline(ctx, eventsChan)
+	errcList = append(errcList, errc)
+
+	eventsChan, errc = t.enrichProcess(ctx, eventsChan)
```

Comment:
@NDStrahilevitz do we want to move this to the processing stage or keep it where the container enrichment is?